### PR TITLE
chore(): change Stacks to Ensono Stacks

### PR DESCRIPTION
**Pull request by isaac-shek**
#### 📲 What
Marketing team feedback to have Ensono Stacks, not just Stacks
Remove Frontend-archived docs

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/stackscli/examples.mdx [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ The command that needs to be run next is displayed at the end of the output from
 
 ### Nx Apps project using the interactive command
 
-To create an empty Stacks Nx Project start by running the interactive command.
+To create an empt Ensono Stacks Nx Project start by running the interactive command.
```
Comment (leesaxby) [inline docs/stackscli/examples.mdx:1241]: Got `empt` here instead of `empty`

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Contributing
 ------------------
 
-Thank you for considering contributing to [Ensono](https://www.ensonodigital.com) Stacks. It's people like you that continue to make Stacks the best it can be.
+Thank you for considering contributing to [Ensono Stacks](https://www.ensonodigital.com). It's people like you that continue to make Ensono Stacks the best it can be.
```
Comment (isaac-shek) [inline .github/CONTRIBUTING.md:4]: Should this URL be [stacks.ensono.com/docs/contributions](https://stacks.ensono.com/docs/contributions)?

**Review by leesaxby — APPROVED** [review]

**Pull request merged**